src/components/Footer/sgds-footer.ts

```diff
@@ -13,36 +13,66 @@ export interface ColumnLinks {
 }
 
 /**
+ * @summary The footer contains supporting information for your service at the bottom of your website. All .gov.sg digital services shall contain a Global Footer Bar across all pages. The Global Footer Bar should include the name of the digital service, contact information, a privacy statement and the terms of use.
+ * 
  * @csspart footer-top - The component's footer-top section container.
  * @csspart footer-bottom - The component's footer-bottom section container.
  */
 @customElement("sgds-footer")
 export class SgdsFooter extends SgdsElement {
   static styles = [SgdsElement.styles, styles];
 
-  @property()
-  title = ``;
-  @property()
-  description = ``;
+  /**
+   * 	Sets title of SgdsFooter
+   */
+  @property({ type: String})
+  title : string;
```

Comment:
Hello, if there is no default value empty string here, then need to handle it below , line 93. Otherwise if user dont use title attribute, title will appear undefined. Same for the other props with default values removed

${this.title ? item.title : "" } 

I wrote with default values, so i didnt handle it during render

Reply:
Revoled in [b8d0d23](https://github.com/GovTechSG/sgds-web-component/pull/61/commits/b8d0d23939f2a727773bfc9a222cd96d21ea97bd)

`new` added prop for copyrightLiner. DSS specific. Added a [feature request](https://github.com/GovTechSG/sgds-govtech-react/issues/152) in React as well